shared/chat/inbox/container/index.tsx

```diff
@@ -169,12 +171,19 @@ const Connected = Container.namedConnect(
     if (!showAllSmallRows) {
       smallTeams = smallTeams.slice(0, stateProps.inboxNumSmallRows)
     }
-    const smallRows = makeSmallRows(smallTeams)
-    const bigRows = makeBigRows(bigTeams)
+    let smallRows = makeSmallRows(smallTeams)
+    let bigRows = makeBigRows(bigTeams)
+    const teamBuilder: Array<RowItemTeamBuilder> = [{type: 'teamBuilder'}]
+    if (smallRows.length !== 0) {
+      if (bigRows.length === 0) {
+        smallRows = [...smallRows, ...teamBuilder]
```

Comment:
change teamBuidler to just the RowItemTeamBuilder and just add it, you don't need to spread it
`smallRows = [...smallRows, teamBuilder]`
or just `smallRows.push(teamBuilder)`